features/openhab-addons-external/pom.xml

```diff
@@ -108,6 +108,7 @@
                 <artifact><file>src/main/resources/conf/panasonictv.cfg</file><type>cfg</type><classifier>panasonictv</classifier></artifact>
                 <artifact><file>src/main/resources/conf/pebble.cfg</file><type>cfg</type><classifier>pebble</classifier></artifact>
                 <artifact><file>src/main/resources/conf/piface.cfg</file><type>cfg</type><classifier>piface</classifier></artifact>
+                <artifact><file>src/main/resources/conf/pilight.cfg</file><type>cfg</type><classifier>piface</classifier></artifact>
```

Comment:
This appears to be causing the build to fail. Copy/Paste error?

````
13:40:15 [ERROR] Failed to execute goal org.codehaus.mojo:build-helper-maven-plugin:1.9.1:attach-artifact (attach-artifact) on project openhab-addons-external: The artifact with same type and classifier: cfg:piface is used more than once. -> [Help 1]
````

https://ci.openhab.org/job/PR-openHAB1-Addons/2415/console